manifests/0000_51_olm_02_operator_clusterrole.yaml

```diff
@@ -219,6 +219,17 @@ rules:
     - get
     - list
     - watch
+  - apiGroups:
+    - olm.operatorframework.io
+    resources:
+    - clustercatalogs
+    verbs:
+    - get
```

Comment:
We probably don't need get permissions since we only ever fetch from the cache.

Reply:
removed in https://github.com/openshift/cluster-olm-operator/pull/73/commits/5b237d4e4d462f19ff7571661b839393bde6b676